scripts/core/superdesk-api.d.ts

```diff
@@ -1901,5 +1901,7 @@ declare module 'superdesk-api' {
         source?: string;
         translations?: {};
         altids?: {[key: string]: string};
+        aliases?: Array<string>;
```

Comment:
Can you add a comment describing the difference between source and original_source?